requirements.txt

```diff
@@ -17,6 +17,7 @@ torch
 transformers==3.3.0
 tensorflow>=2
 tensorflow_hub
+tensorflow_text>=2
```

Comment:
Since `tensorflow_text` is not needed, could you remove this from the requirements.txt?

Reply:
If you remove it there will be problems with Tensorflow:
`Op type not registered 'SentencepieceOp'`
See https://github.com/tensorflow/tensorflow/issues/38597 for an example

Reply:
Yes I just also realized this while trying to run an attack with MUSE. Please ignore the previous comment.